example/material_tree_example/lib/src/material_tree_nested_component_rendering_demo.dart

```diff
@@ -40,6 +40,8 @@ import 'package:angular_components/model/ui/has_factory.dart';
     ''',
   styleUrls: ['material_shadow.scss.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
+  // TODO(google): Change preserveWhitespace to false to improve codesize.
+  preserveWhitespace: true,
```

Comment:
It is actually better to not use preserveWhitespace. If you see it in code if is because the code is older and we haven't been able to migrate it.